okhttp/src/main/java/com/squareup/okhttp/internal/Internal.java

```diff
@@ -35,6 +38,12 @@
  */
 public abstract class Internal {
   public static final Logger logger = Logger.getLogger(OkHttpClient.class.getName());
+
+  public static void initializeInstanceForTests() {
```

Comment:
This is quite unfortunate. Seems like tests shouldn't need to know about `Internal` !